src/plugins/order_stash.coffee

```diff
@@ -176,31 +181,36 @@ class OrderStash
     font-style: italic;
     line-height: 13px;
     box-sizing: border-box;
+
+    transition: none;
```

Comment:
I did not add this one :smile: it was already defined here: https://github.com/skroutz/analytics.js/pull/20/files#diff-7316f697d60038fadbc42b52a47928d0L164